### PR TITLE
Updated some deps and tested with iojs, and a small speling correction.

### PR DESCRIPTION
**Pull request by wolfeidau**
Working like a treat, with the babel loader added.

Happy to discuss iojs stuff as it isn't everyones cup of tea atm. Sadly the jtangelder/sass-loader module hasn't released a new version since adding support.

No rush, and always happy to accept critique.

Cheers

### CODE REVIEW

File: app/index.jsx [hunk 1]
```diff
@@ -49,11 +47,11 @@ var routes = (
   </Router.Route>
 );
 
-
-Router.run(routes, (Handler, state) => {
+// use the HTML5 history API for cleaner URLs
+Router.run(routes, Router.HistoryLocation, (Handler, state) => {
```
Comment (nelix) [inline app/index.jsx]: Can you add --history-api-fallback to start.sh so this works on reload?

File: app/index.jsx [hunk 2]
```diff
@@ -49,11 +47,11 @@ var routes = (
...
   log('route change', state);
 
   // store logic here if required
 
   React.render(<Handler/>, document.body);
-});
+});
```
Comment (nelix) [inline app/index.jsx]: Is this something I can fix with .editorconfig? I like trailing new lines

**Comment by nelix** [comment]
This is good, thanks @wolfeidau

File: app/index.jsx (hunk 1 above)
Comment (grapegravity) [inline app/index.jsx]: The comment here seems unnecessary to me, honestly

File: app/index.jsx (hunk 2 above)
Comment (grapegravity) [inline app/index.jsx]: That’s probably not a bad call. Choose some settings and also refer to editorconfig in docs about contributing?

**Comment by wolfeidau** [comment]
As requested I added the switch to the start.sh, removed the comment and restored the new line.

Cheers for the feedback.

**Comment by nelix** [comment]
Thanks a lot! :+1:

**Pull request merged**